Examples/timberman.md

```diff
@@ -0,0 +1,72 @@
+# Timberman
+
+**Playstore Link:**  [Timberman](https://play.google.com/store/apps/details?id=com.dm.timber&hl=en)
+
+##### About the game:
+Timberman is an oldschool arcade style casual game. You have to put on your checkered shirt, pull up your sleeves, pick up the sharpened ax and ... grow a beard. In a moment there will be splinters flying and knocks off ax will bear echoed through the forest. Show how quick and how efficiently you can cut down a hundred-meter tall pine tree. But be careful, work of a lumberjack is tough and it's not hard for the accident at work. In addition, time is chasing you in every step not giving even a moment to rest. 
+
+##### Difficulty Level : 
+Easy
+
+##### Overview
```

Comment:
This needs to be h3 and not h5. Use three ###. To learn more, see the example: https://raw.githubusercontent.com/GameAutomators/eBook-Source/master/Examples/05-ready-steady-bang.md